CHANGES.md

```diff
@@ -33,6 +35,8 @@ To be released.
  -  `BlockChain<T>.MineBlock()` and `BlockChain<T>.GetNextTxNonce()` methods
     became to ignore transactions that didn't follow `Transaction<T>.Nonce`
     sequentially and treat them as pendings.  [[#365]]
+ - `BlockChain<T>` became to evaluate `IBlockPolicy<T>.BlockAction` and set the
+   state when a block is appended to the chain.  [[#319], [#367]]
```

Comment:
it looks like the link about #319 is missing.

Reply:
Added!